li-apache-kafka-clients/build.gradle

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2017 LinkedIn Corp.
+ * Licensed under the BSD 2-Clause License (the "License"). 
+ * See License in the project root for license information.
+ */
+import com.linkedin.gradle.build.DistributeTask
+
+plugins {
+    id "java-library"
+    id "maven-publish"
+    id "com.jfrog.artifactory"
+    id "idea"
+}
+
+dependencies {
+    compile "org.apache.kafka:kafka-clients:${rootProject.ext.kafkaVersion}"
+}
+
+publishing {
+    publications {
+        java(MavenPublication) {
+            from components.java
```

Comment:
Minor duplication that you can wrap if you push out the common logic to script plugin applied with "apply from:"

Reply:
beyond my gradle knowledge. would you be ok with submitting a follow-up PR ?